cmd/tendermint/commands/rollback_test.go

```diff
@@ -0,0 +1,61 @@
+package commands_test
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/tendermint/tendermint/cmd/tendermint/commands"
+	"github.com/tendermint/tendermint/rpc/client/local"
+	rpctest "github.com/tendermint/tendermint/rpc/test"
+	e2e "github.com/tendermint/tendermint/test/e2e/app"
+)
+
+func TestRollbackIntegration(t *testing.T) {
+	dir := t.TempDir()
+	cfg, err := rpctest.CreateConfig(t.Name())
+	require.NoError(t, err)
+	cfg.BaseConfig.DBBackend = "goleveldb"
```

Comment:
why do we choose this one?

Reply:
it's the default. Do you have a better suggestion?

Reply:
badger

Reply:
mushroom

Reply:
but badger doesn't come straight out the box like goleveldb does. Won't I need to do some import magic?

Reply:
Yeah, I think you'd need to be sure your tests build with the `badgerdb` tag since that's how `tm-db` selects which backends to include.